cawg_identity/src/claim_aggregation/ica_validation_error.rs

```diff
@@ -59,9 +59,31 @@ pub enum IcaValidationError {
     #[error("unsupported issuer DID ({0})")]
     UnsupportedIssuerDid(String),
 
-    /// Issue date is missing.
-    #[error("credential does not have a valid_from date")]
+    /// DID could not be resolved (network error, etc.).
+    #[error("DID could not be resolved ({0})")]
+    DidResolutionError(String),
+
+    /// Invalid issuer DID document.
+    #[error(
+        "The DID document could not be parsed or did not contain usable public key material ({0})"
+    )]
+    InvalidDidDocument(String),
+
+    /// RFC 3161 time stamp is invalid.
+    #[error("The RFC 3161 time stamp was not valid for this credential")]
+    InvalidTimeStamp,
+
+    /// `validFrom` date is missing.
+    #[error("credential does not have a validFrom date")]
```

Comment:
Why do some error messages start with an upper case and others don't?

Reply:
Thanks for the catch. Made these consistent (lowercase unless starting with an acronym).